File: vue/src/api/dsl-history.js
import request from '@/utils/request'

const api = '/api/dslHistory/'

export function CleanAction(data) {
  return request({
    url: api + `CleanAction`,
    method: 'post',
    data
  })
}

export function ListAction(data) {
  return request({
    url: api + `ListAction`,
    method: 'post',
    data
  })
}

File: pkg/web/dsl_history.go
package web

// DslHistory 路由
func (this *WebServer) runDslHistory() {

	const AbsolutePath = "/api/dslHistory"
	dslHistory := this.engine.Group("ES查询历史记录", AbsolutePath)
	{
		dslHistory.POST(false, "清空DSL查询历史记录", "/CleanAction", this.dslHistoryController.CleanAction)
		dslHistory.POST(false, "查看DSL查询历史记录", "/ListAction", this.dslHistoryController.ListAction)
	}
}
